### ISSUE TITLE
Not woking android browser.

### ISSUE THREAD

**Issue by ashishsharma243** [blasten/turn.js#67]
Your project is looking good. And I need this type of feature on my project but it is not working on any android browser from 2.2 to 4.0 version.If you give me updated code which run on android browser also much like iPhone.

Please I need this feature.

Thanks in advance

**Comment by blasten** [comment]
Yes, I have that in my schedule. I tested on Chrome for android and it worked, but I'm aware of performance problems.

**Comment by abhi001** [comment]
I have created a magazine using latest turn.js and "double tap zoom" feature is not working on android default browser. I have tested it on android version 4.0.3 and 4.1.2. 
Is there any update for android default browser ?

**Comment by blasten** [comment]
@abhi001 Yes, I will show the improvements shortly.

**Comment by cinder92** [comment]
hi there!

is any update for turn.js 3rd for android default browser? i tested o a tablet with android 4.0 ice creame and is not working the page are broken
